ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/manager/EthServerTest.java

```diff
@@ -68,6 +78,14 @@ public void shouldHandleDataBeingUnavailableWhenRespondingToNodeDataRequests() t
 
   @Test
   public void shouldLimitNumberOfResponsesToNodeDataRequests() throws Exception {
+    new EthServer(
+        blockchain,
+        worldStateArchive,
+        ethMessages,
+        EthServer.DEFAULT_REQUEST_LIMIT,
+        EthServer.DEFAULT_REQUEST_LIMIT,
+        EthServer.DEFAULT_REQUEST_LIMIT,
+        EthServer.DEFAULT_REQUEST_LIMIT);
```

Comment:
Not sure how this is testing is passing with the default limit at 200.  The intention of this test is to check that the server caps the amount of data returned.